specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```
@@ -1474,11 +1512,11 @@
         },
         "groupFileURI": {
           "type": "string",
-          "description": "The URI of the file containing the group information (in etc/group file format).  This field must be populated when 'usernameSource' is set to 'File'."
+          "description": "The URI of the file containing the group information (in /etc/group file format).  This field must be populated when 'usernameSource' is set to 'File'."
```

Super nitpicky, but while I'm here and since I have other comments in this review.
```suggestion
          "description": "The URI of the file containing the group information (in /etc/group file format). This field must be populated when 'usernameSource' is set to 'File'."
```

---

Fixed in next commit.